src/styles/theme.ts

```diff
@@ -1,4 +1,4 @@
-export default {
+export const defaultTheme = {
```

Comment:
I think that it's better, in favor of consistency, to name the theme variable according to the filename.

```suggestion
export const theme = {
```